src/Command/CheckCommand.php

```diff
@@ -139,7 +139,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 'ignoreErrors' => [],
                 'drupal' => [
                     'drupal_root' => $this->drupalRoot,
-                ]
+                ],
+                'tmpDir' => sys_get_temp_dir() . '/phpstan' . '_' . getmypid(),
```

Comment:
I don't know if we should be modifying the `tmpDir` where PHPStan puts its caches. If this is having issues in Jenkins on a shared environment, is there a way to customize the system temp directory PHP pulls?